config/config.go

```diff
@@ -429,36 +435,38 @@ func (cfg Config) BlockchainAddress() (address.Address, error) {
 }
 
 // KeyPair returns the decoded public and private key pair
-func (cfg Config) KeyPair() (keypair.PublicKey, keypair.PrivateKey, error) {
+func (cfg Config) KeyPair() (*ecdsa.PublicKey, *ecdsa.PrivateKey, error) {
 	pk, err := keypair.DecodePublicKey(cfg.Chain.ProducerPubKey)
 	if err != nil {
-		return keypair.ZeroPublicKey,
-			keypair.ZeroPrivateKey,
-			errors.Wrapf(err, "error when decoding public key %s", cfg.Chain.ProducerPubKey)
+		return nil, nil, errors.Wrapf(err, "error when decoding public key %s", cfg.Chain.ProducerPubKey)
 	}
 	sk, err := keypair.DecodePrivateKey(cfg.Chain.ProducerPrivKey)
 	if err != nil {
-		return keypair.ZeroPublicKey,
-			keypair.ZeroPrivateKey,
+		return nil,
```

Comment:
two nill in one line?

Reply:
Yes. In the case of error, both public key and private key are returned as nil